cranelift/wasm/src/code_translator.rs

```diff
@@ -73,13 +73,27 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             builder.set_val_label(val, label);
         }
         Operator::LocalSet { local_index } => {
-            let val = state.pop1();
+            let mut val = state.pop1();
+
+            // ensure SIMD values are cast to their default Cranelift type, I8x16
```

Comment:
Please capitalize the first word of a sentence in a comment, and end it with a period if appropriate, as it is here.